src/app/pages/quickMaps/pages/dietaryChange/components/options/options.component.scss

```diff
@@ -1,19 +1,126 @@
 @import 'src/assets/scss/_colour';
 
 .container {
-  display: flex;
   margin-bottom: 10px;
   border: 1px solid $color_primary;
   border-radius: 5px;
+  align-items: center;
+  background-color: rgb(252, 250, 250);
+  display: flex;
+  width: 100%;
   flex-flow: row wrap;
-  background-color: white;
   align-items: center;
-  justify-content: center;
+  justify-content: space-between;
+  font-size: 18px;
+  gap: 5px;
+
+  & .wideItem {
+    flex-grow: 1;
+    display: flex;
+    margin: 1em 0.5em;
+  }
+
+  & .right-text {
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: flex-start;
+  }
+
+  & .form,
+  & .left-text {
+    flex-flow: row wrap;
+  }
+
+  & .form {
+    align-items: center;
+    justify-content: space-around;
+  }
+
+  & .left-text {
+    align-items: flex-start;
+    justify-content: space-around;
+  }
+
+  & .left-text > div {
+    margin: 0em 0.5em;
+  }
+}
+
+//set radio buttons to  primary colour and make label bold on selection
+::ng-deep .mat-radio-checked .mat-radio-label-content {
+  font-weight: bold;
+  color: $color_primary;
+}
+
+::ng-deep.mat-radio-button.mat-accent.mat-radio-checked .mat-radio-outer-circle,
+.mat-radio-outer-circle {
+  border-color: $color_primary !important;
```

Comment:
!important is a seriously last resort.  Reference a class in the template to get around having to use it?